packages/zowe-explorer/src/webviews/vite.config.ts

```diff
@@ -40,6 +41,18 @@ export default defineConfig({
         checker({
             typescript: true,
         }),
+        viteStaticCopy({
+            targets: [
+                {
+                    src: "../node_modules/@vscode/codicons/dist/codicon.css",
+                    dest: "codicons/",
+                },
+                {
+                    src: "../node_modules/@vscode/codicons/dist/codicon.ttf",
+                    dest: "codicons/",
+                },
+            ],
+        }),
```

Comment:
Nice! We should do this for the following PR too:
- #2925 

Thanks Timothy for point me to this, and thanks to you (Trae) for implementing it  🙏🏽

Reply:
I initially had a hacky way of injecting the codicons into the VSIX but Timothy proposed the static copy approach - thanks for that Timothy 😋